code/modules/robotics/bot/secbot.dm

```diff
@@ -416,6 +664,24 @@ Report Arrests: <A href='?src=\ref[src];operation=report'>[report_arrests ? "On"
 				src.mover = null
 			src.frustration = 0
 		return
+		
+	proc/shootgun_attack(var/mob/living/carbon/M)
+		src.icon_state = "secbot-c[src.emagged >= 2 ? "-wild" : null]"
+		if (src.emagged >= 2)
+			shotcount = burst * rand(2,3)
+		else
+			shotcount = burst
+		last_attack = world.time
+
+		while (shotcount > 0 && src.target)
+			shoot_projectile_ST_pixel_spread(src, proj, src.target, 0, 0 , spread)
+			shotcount--
+			sleep(refire)
```

Comment:
```suggestion
		shoot_projectile_ST_pixel_spread(src, proj, get_turf(src.target), 0, 0 , spread)
```
Fire at target's turf instead of target, use shoot proc properly